src/ResourceManager/PowerBI/Commands.PowerBI/Commands/RemoveAzureRmPowerBIEmbeddedCapacity.cs

```diff
@@ -0,0 +1,111 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System;
+using System.Management.Automation;
+using System.Security.Permissions;
+using Microsoft.Azure.Commands.PowerBI.Models;
+using Microsoft.Azure.Commands.PowerBI.Properties;
+using Microsoft.Azure.Commands.PowerBI.Utilities;
+using Microsoft.Azure.Management.PowerBIDedicated.Models;
+using Microsoft.Rest.Azure;
+
+namespace Microsoft.Azure.Commands.PowerBI
+{
+    [Cmdlet(VerbsCommon.Remove, "AzureRmPowerBIEmbeddedCapacity", SupportsShouldProcess = true), 
```

Comment:
@selasagi this cmdlet should have a [`DefaultParameterSetName`](https://github.com/Azure/azure-powershell/wiki/PowerShell-Cmdlet-Design-Guidelines#defaultparametersetname) that is `CmdletParameterSet` so the user will be prompted to provided a value for `Name` and `ResourceGroupName` if they don't provide any parameters.

Reply:
added